llama-index-integrations/llms/llama-index-llms-sambanova/pyproject.toml

```diff
@@ -21,9 +21,9 @@ ignore_missing_imports = true
 python_version = "3.8"
 
 [tool.poetry]
-authors = ["Your Name <you@example.com>"]
-description = "llama-index llms sambanova cloud integration"
-name = "llama-index-llms-sambanovacloud"
+authors = ["Rodrigo Maldonado <rodrigo.maldonado@pucp.edu.pe>"]
+description = "llama-index llms sambanova cloud and sambastudio integration"
+name = "llama-index-llms-sambanova"
```

Comment:
We cannot publish under this name, as someone published something else under this name. This is why everything was renamed to samba nova cloud

Reply:
(see above comment)